adr/0002-API-params-naming.md

```diff
@@ -0,0 +1,27 @@
+# 1. Record architecture decisions
+
+Date: 2020-02-06
+
+## Status
+
+Accepted
+
+## Context
+
+The naming of the params used in the API needs be consistent.
+
+For example:
+nomis_offender_id, prison_number and nomis_prison_number refer to the same data.
+
+## Decision
+Use param names similar to those used on [DPS APIs](https://gateway.prod.nomis-api.service.hmpps.dsd.io/elite2api/swagger-ui.html#//offenders/getAddressesByOffenderNoUsingGET)
+
+Use snake_case in place of camelCase.
+Consider to prefix the param with `nomis_` to clarify the source.
+
+For example:
+`offenderNo` -> `nomis_offender_no`
```

Comment:
Unfortunately I believe this is the opposite of the decision that we have actually taken. I think we have decided to use prison_number for this field throughout.

Reply:
and actually the word 'consider' implies an optional behaviour rather than a decision?

Reply:
Hi @starswan this is a new decision to introduce consistency with DPS and remove an ambiguous identifier. As a result of this decision, we will change all references to `prison_number` (except if/where it's published in our Swagger docs).

Reply:
@starswan I'll remove consider, it is  ambiguous

Reply:
I am going to merge this, @starswan in case we'll find cases that for some reasons cannon use this convention, we can amend the ADR.